draft-ietf-tls-tls13.md

```diff
@@ -3124,14 +3141,17 @@ first.
 
 Any ticket MUST only be resumed with a cipher suite that has the
 same KDF hash as that used to establish the original connection,
-and if the client provides the same SNI value as described in
-Section 3 of {{RFC6066}}.
+and only if the client provides the same SNI value as in the original
+connection, as described in Section 3 of {{RFC6066}}.
 
 Note: Although the resumption master secret depends on the client's second
 flight, servers which do not request client authentication MAY compute
 the remainder of the transcript independently and then send a
 NewSessionTicket immediately upon sending its Finished rather than
-waiting for the client Finished.
+waiting for the client Finished.  This might be appropriate in cases
+where the client is expected to open multiple TLS connections in
+parallel and would benefit from the reduced overhead of a resumption
+handshake, for example.
```

Comment:
Thanks for this text. We had a sketchier reason for this, so it's nice to have a really solid one. Also, this is a really nice performance benefit of TLS 1.3 over TLS 1.2 with H1, b/c in TLS 1.2 the NST comes in the server's second flight, even with False Start.